apps/application-system/api/src/app/modules/application/e2e/utils.ts

```diff
@@ -0,0 +1,23 @@
+import request, { CallbackHandler } from 'supertest'
+
+export const getRequestMethod = (
```

Comment:
I saw that you added this to `@island.is/testing/nest` so you should remove this file.